spec/models/user_spec.rb

```diff
@@ -315,12 +315,12 @@
         it { is_expected.not_to be_able_to(:read, file) }
         it { is_expected.to be_able_to(:read, tile) }
         it { is_expected.not_to be_able_to(:read, image) }
-        it { is_expected.not_to be_able_to(:stream, media) }
+        it { is_expected.to be_able_to(:stream, media) }
```

Comment:
We think this was copy-pasta with a rights scenario that wasn't accounted for, and reviewed the change with @LynnMcRae.. In short, we believe `no-download` resources should be streamable (but not downloadable).